internal/provider/config_resource_test.go

```diff
@@ -61,24 +61,52 @@ func TestAccConfigWithoutBasicAuthResource(t *testing.T) {
 	})
 }
 
+func TestAccConfigWithCacheMaxAgeResource(t *testing.T) {
+	var TEST_ENVIRONMENT_ID = randomString(10)
+	var INITIAL_HOST = "www.thgaltitude.com"
+	var CACHE_MAX_AGE = "360"
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { testAccPreCheck(t) },
+		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccKVResourceConfigCacheMaxAge(TEST_ENVIRONMENT_ID, INITIAL_HOST, ""),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("altitude_mte_config.cache-field-test", "config.routes.0.host", INITIAL_HOST),
+					resource.TestCheckResourceAttr("altitude_mte_config.cache-field-test", "environment_id", TEST_ENVIRONMENT_ID),
+					resource.TestCheckNoResourceAttr("altitude_mte_config.cache-field-test", "config.routes.0.cache_max_age"),
+				),
+			},
+			{
+				Config: testAccKVResourceConfigCacheMaxAge(TEST_ENVIRONMENT_ID, INITIAL_HOST, CACHE_MAX_AGE),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("altitude_mte_config.cache-field-test", "config.routes.0.host", INITIAL_HOST),
+					resource.TestCheckResourceAttr("altitude_mte_config.cache-field-test", "environment_id", TEST_ENVIRONMENT_ID),
+					resource.TestCheckResourceAttr("altitude_mte_config.cache-field-test", "config.routes.0.cache_max_age", CACHE_MAX_AGE),
+				),
+			},
+		},
+	})
+}
+
 func testAccKVResourceConfigWithBasicAuth(environmentId string, host string) string {
 	return fmt.Sprintf(`
 resource "altitude_mte_config" "tester" {
   config = {
     routes = [
       {
-        host                 = "%s"
-        path                 = "/test"
-        enable_ssl           = true
-        preserve_path_prefix = true
-				shield_location			 = "London"
+		host                 = "%s"
+		path                 = "/test"
+		enable_ssl           = true
+		preserve_path_prefix = true
+		shield_location		 = "London"
```

Comment:
Really small thing, in this test file please make all the formatting in the strings align